interface/patient_file/merge_patients.php

```diff
@@ -313,27 +483,49 @@ function mergeRows($tblname, $colname, $source_pid, $target_pid)
    <input type='hidden' name='form_source_pid' value='0' />
   </td>
   <td>
-    <?php echo xlt('This is the chart that is to be merged into the main chart and then deleted.'); ?>
+    <?php echo xlt(
+        'This is the chart that is to be merged into the main chart 
+        and then deleted.'
```

Comment:
ensure don't break up translation constant into multiple line. need to make these one line no matter how long they are (or else will break in the translation pipeline)

Reply:
still needs to be addressed